qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1471,6 +1490,11 @@ def schedule_information(self) -> Tuple[List[str], List[Dict]]:
 
         return ["name", "qubits", "schedule"], schedules
 
+    @deprecate_func(
+        since="0.6",
+        additional_msg="Loading file now retrieves full calibration data including parameters.",
```

Comment:
```suggestion
        additional_msg="Loading and saving calibrations in CSV format is deprecated.",
```
I think we should just state outright that we deprecate CSV. this will make the transition clearer.